CDTDatastore/CDTDatastore.m

```diff
@@ -451,6 +451,24 @@ - (BOOL)ensureDatabaseOpen
 {
     return [_database openWithEncryptionKeyProvider:self.keyProvider];
 }
+/**
+ * Validates that attachments are keyed by attachment name.
+ * @param attachments The attachments dictionary to validate.
+ * @return true if the attachments dictionary is of the form @[ attachment.name : attachment ] false
```

Comment:
Small fix to capitalize True and possibly add comma:
`True if the attachments dictionary is of the form @[ attachment.name : attachment ], and false otherwise.`